### PR TITLE
Fix invalid characters in kafka metric names

### PR DESCRIPTION
**Pull request by avishai-ish-shalom**

### CODE REVIEW

File: src/collectors/kafkastat/kafkastat.py [hunk 1]
```diff
@@ -128,7 +128,8 @@ def query_mbean(self, objectname, key_prefix=None):
                         key_prefix = key_prefix + '.' + key
 
         metrics = {}
-
+        trans_table = string.maketrans(', ','__')
```
Comment (bmhatfield) [inline src/collectors/kafkastat/kafkastat.py]: Silly question, but why two `_` characters instead of 1?

Comment (bmhatfield) [inline src/collectors/kafkastat/kafkastat.py]: Ah, just read the maketrans doc more carefully. This all looks good to me.

**Comment by bmhatfield** [comment]
Thanks!

File: src/collectors/kafkastat/kafkastat.py (hunk 1 above)
Comment (jaingaurav) [inline src/collectors/kafkastat/kafkastat.py]: Seems this change broke travis tests:

https://travis-ci.org/python-diamond/Diamond/builds/108056178

**Pull request merged**